### PR TITLE
Add k8s manifests

### PR DESCRIPTION
**Pull request by rasheedamir**
add k8s manifests

### CODE REVIEW

File: kubernetes-manifests/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    app: restful-distributed-lock-manager
+    group: com.stakater.platform
+    version: 0.5.1
+    provider: stakater
+  name: ingress-monitor-controller
```
Comment (waseem-h) [inline kubernetes-manifests/deployment.yaml]: Shouldn't this be changed?

File: kubernetes-manifests/manifests.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      provider: stakater
+- apiVersion: extensions/v1beta1
+  kind: Deployment
+  metadata:
+    labels:
+      app: restful-distributed-lock-manager
+      group: com.stakater.platform
+      version: 0.5.1
+      provider: stakater
+    name: ingress-monitor-controller
```
Comment (waseem-h) [inline kubernetes-manifests/manifests.yaml]: Need to change this as well

File: kubernetes-manifests/service.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: restful-distributed-lock-manager
+    group: com.stakater.platform
+    version: 0.5.1
+    provider: stakater
+  name: app
```
Comment (waseem-h) [inline kubernetes-manifests/service.yaml]: Shouldn't we name the deployment and service to like "restful-distributed-lock-manager"?

**Review by waseem-h — CHANGES_REQUESTED** [review]

File: kubernetes-manifests/deployment.yaml (hunk 1 above)
Comment (rasheedamir) [reply kubernetes-manifests/deployment.yaml]: good catch!

File: kubernetes-manifests/manifests.yaml (hunk 2 above)
Comment (rasheedamir) [reply kubernetes-manifests/manifests.yaml]: good catch!

File: kubernetes-manifests/service.yaml (hunk 3 above)
Comment (rasheedamir) [reply kubernetes-manifests/service.yaml]: good catch!

**Review by waseem-h — APPROVED** [review]

**Pull request merged**